Classes/Providers/OTT/Services/PhoenixAPIDefines.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PhoenixAPIDefines.swift
+//  Pods
+//
+//  Created by Rivka Peleg on 02/03/2017.
+//
+//
+
+import Foundation
+
+public enum AssetType: String {
+    case media = "media"
+    case epg = "epg"
+}
+
+public enum PlaybackContextType: String {
```

Comment:
for this to be visible in objc same we need to make it same structure as old `AssetType`, @objc modifier should be added and the type should be Int, for the string values create a computed property.